pom.xml

```diff
@@ -9,9 +9,9 @@
 
     <name>ecoCode iOS plugin</name>
     <description>Help the earth, adopt this green plugin for your applications</description>
-    <inceptionYear>2022</inceptionYear>
+    <inceptionYear>2023</inceptionYear>
     <organization>
-        <name>Green code Initiative</name>
+        <name>green-code-initiative</name>
         <url>https://collectif.greenit.fr</url>
```

Comment:
change also organization URL please

Reply:
With this one :  https://github.com/green-code-initiative ?

Reply:
No it's fine !
I do it !